workspace/src/main/openapi/workspace.yaml

```diff
@@ -1066,6 +1088,8 @@ components:
           defaultRunTemplateDataset:
             run1: "data1"
         datasetCopy: true
+        creationDate: 1649941200000
+        lastUpdate: 1649941200000
```

Comment:
This should be removed and `creationInfo` / `updateInfo` should be added instead